azure-javascript/index.js

```diff
@@ -3,16 +3,15 @@ const pulumi = require("@pulumi/pulumi");
 const azure = require("@pulumi/azure");
 
 // Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
-    location: "WestUS",
-});
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
 
 // Create an Azure resource (Storage Account)
 const account = new azure.storage.Account("storage", {
+    // The location for the storage account will be derived automatically from the resource group.
     resourceGroupName: resourceGroup.name,
     accountTier: "Standard",
     accountReplicationType: "LRS",
 });
 
 // Export the connection string for the storage account
-exports.connectionString = account.primaryConnectionString;
+exports.connectionString = account.primaryConnectionString;
```

Comment:
Trailing newline?

Reply:
My VS Code formatter removed the new-line when I saved the file. Fixed it now.